### PR TITLE
Do not start consensus when no tips available

### PR DESCRIPTION
**Pull request by marcinwadon**

### CODE REVIEW

**Review by tbekas — DISMISSED** [review]
LGTM, but please wait for the approval of @TheMMaciek

File: modules/dag-l1/src/main/scala/org/tessellation/dag/l1/StateChannel.scala
Comment (TheMMaciek) [inline modules/dag-l1/src/main/scala/org/tessellation/dag/l1/StateChannel.scala:189]: Not necessary as it will change with the download/redownload in L1 PR

Comment (marcinwadon) [reply modules/dag-l1/src/main/scala/org/tessellation/dag/l1/StateChannel.scala:189]: Ok, so you'll remove that

File: modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/consensus/block/Validator.scala
Comment (TheMMaciek) [inline modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/consensus/block/Validator.scala:63]: Maybe one log message with all conditions? Like: "Cannot start own consensus round! noRoundInProgress: $noOwnRoundInProgress --- stateReadyForConsensus: $stateReadyForConsensus ..."

**Review by TheMMaciek — APPROVED** [review]

**Pull request merged**
